Update azure-stack-acs-differences.md

GeorgeH-MS opened the pull request:
Explain live snapshot (snapshot copy a page blob which is used by a living VM as unmanaged disk) is not supported. Or at least we don't guarantee the snapshot is consist with the page blob at the time snapshot is made.

azurecla commented:
Hi __@cpswan__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

telmosampaio commented:
I already made these changes, and others this morning. Please close this PR.

GeorgeH-MS committed:
Update azure-stack-acs-differences.md

Explain live snapshot (snapshot copy a page blob which is used by a living VM as unmanaged disk) is not supported. Or at least we don't guarantee the snapshot is consist with the page blob at the time snapshot is made.
